cloudflare/resource_cloudflare_ipsec_tunnel.go

```diff
@@ -0,0 +1,128 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareIPsecTunnel() *schema.Resource {
+	return &schema.Resource{
+		Schema: resourceCloudflareIPsecTunnelSchema(),
+		Create: resourceCloudflareIPsecTunnelCreate,
+		Read:   resourceCloudflareIPsecTunnelRead,
+		Update: resourceCloudflareIPsecTunnelUpdate,
+		Delete: resourceCloudflareIPsecTunnelDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareIPsecTunnelImport,
+		},
+	}
+}
+
+func resourceCloudflareIPsecTunnelCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*cloudflare.API)
+	client.AccountID = d.Get("account_id").(string)
```

Comment:
this will now need to update that we are not using the api.AccountID field and instead, pull it from the schema.

Reply:
Done, I hope I didn't miss any. I am actually not sure, how does meta behave when run in parallel on different resources with different account IDs? I think meta is not really protected from this, right?

Reply:
i don't _think_ so but i'd need to confirm the internals. it shouldn't be an issue though as in the Tenant use case, the client will be the same (authn/z) but the URLs will need to swap based on the account.